lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1029,6 +1029,9 @@ fc_extras
         try:
             iris.unit.as_unit(attr_units)
         except ValueError:
+        # formatting a unicode string causes an exception. Test for that.
+            if isinstance(attr_units, unicode):
+                attr_units = attr_units.encode('ascii', errors='backslashreplace')
```

Comment:
The reason for the change was to escape any unicode characters - because not doing so causes any string formatting function to except. I could set the 'invalid_units' value to a unicode string as well, it seems odd to have a unicode string where only ASCII characters are used. That's the easy option, anyhow.

Unicode is technically permitted in variables other than CF conventions (standard_name, unit, etc.) so it seems like escaping this isn't the ideal solution anyway - but it would require a chunk of work to ensure no 's.format' or other incompatible operations are attempted on the value. What do you think is the better way for this bug?